composer.json

```diff
@@ -19,7 +19,7 @@
     "prefer-stable": true,
     "require": {
         "php": "^5.3.9|~7.0",
-        "symfony/framework-bundle": "^2.3.27|^2.6.6|~3.0",
```

Comment:
Instead of conflicting, why didn't you use `~2.3.27|~2.6.6|~3.0`. Afaik, require rules are much better than conflict rules and it also is more readable imo